app/models/concerns/hud_reports/length_of_stays.rb

```diff
@@ -70,6 +70,32 @@ module HudReports::LengthOfStays
       (end_date - enrollment.first_date_in_program).to_i
     end
 
+    # Given the reporting period, how many distinct bed nights does the client have?
+    # For entry/exit this is span, for night by night, this is service days.
+    private def bed_nights(enrollment)
```

Comment:
I feel like this should get delegated down to `ServiceHistoryEnrollment` eventually.  It could take optional start and end dates...